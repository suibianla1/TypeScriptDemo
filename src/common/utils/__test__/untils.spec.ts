import { createToken, md5Decode } from '..';
import crypto from 'crypto';
import jwt from 'jsonwebtoken';

jest.mock('jsonwebtoken', () => {
  const sign = () => '';
  return { sign };
});

describe('utils', () => {
  it('createToken', () => {
    expect(createToken({ username: '' })).toBe('');
  });

  it('md5Decode', () => {
    expect(md5Decode('123456')).toMatchSnapshot();
  });
});
