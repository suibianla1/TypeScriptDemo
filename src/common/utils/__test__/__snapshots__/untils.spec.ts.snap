// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`utils md5Decode 1`] = `"e10adc3949ba59abbe56e057f20f883e"`;
